### ISSUE TITLE
[Bug]: Both of the demo templates on the Getting Started page are broken.

### ISSUE THREAD

**Issue by a2800276** [materializecss/materialize#492]
### Before submitting...

- [X] I have searched for [duplicate or closed issues](https://github.com/materializecss/materialize/issues?utf8=%E2%9C%93&q=is%3Aissue).
- [X] I have read the [CONTRIBUTING document](https://github.com/materializecss/materialize/blob/master/CONTRIBUTING.md) and my issue is following the guidelines.
- [X] I have read the template completely before filling it in.

### Context

Trying to load the demo pages of the [Starter](https://materializeweb.com/templates/starter-template/preview.html) and [Parallax](https://materializeweb.com/templates/parallax-template/preview.html) Templates linked to on the "[Getting Started](https://materializeweb.com/getting-started.html)" page.

Mozilla Firefox 128.0.3 / Linux (don't think this is Firefox or OS specific)

### Current Behavior

Both are broken.

![image](https://github.com/user-attachments/assets/03b9871b-29d0-4072-b029-d5ff6da6e97e)

### Expected behavior

They should work.

### Possible Solutions or Causes

The assets they link to all resolve to 404. I assume something is off in your CI:

![image](https://github.com/user-attachments/assets/5591a125-6553-4ffd-bfff-3f4afd92f8ef)

### Steps to reproduce

1.) Open the link to the  [Starter Template](https://materializeweb.com/templates/starter-template/preview.html) 

### Your Environment

* Version used:
* Browser Name and version: Mozilla Firefox 128.0.3
* Operating System and version (desktop or mobile): Desktop Linux
* Additional information you want to tell us: this bug report process might be overkill in some cases :)

**Comment by a2800276** [comment]
The links to the template zips also resolve to 404.

**Comment by wuda-io** [comment]
Yes thats right, they were moved to the documentation repo in 2.1.0. so they have to be rewritten.

**Comment by wuda-io** [comment]
If it is still not fixed, please reopen the issue in the website repo. thanks
